LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2023, the Dart project authors.
+Copyright 2024, the Dart project authors.
```

Comment:
nit: I think it's OK to keep this one as 2023

Reply:
Done.